### PR TITLE
Bug/mustache binding extra space

### LINKED ISSUE #5134: [Bug] Table data is not populating when extra space inside the mustache binding
Issues occurs when extra space is provided in mustache binding:

```
{{
currentRow.dob == null ? "no birth" : moment(currentRow.dob).add(24, "years") < moment() ? "small" : moment(currentRow.dob).add(18, "years") < moment() ? "very small" : "very very small"
}}
```
Worked fine for:
`{{currentRow.dob == null ? "no birth" : moment(currentRow.dob).add(24, "years") < moment() ? "small" : moment(currentRow.dob).add(18, "years") < moment() ? "very small" : "very very small"}}
`

- Version: Cloud 
- OS: MacOSX
- Browser: chrome
- Environment: production

### PR DESCRIPTION
**Pull request by techbhavin**
Fixes #5134 

## Type of change

- Bugfix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ]  I have commented my code, particularly in hard-to-understand areas
- [ ]  I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:red_circle: Total coverage has decreased</summary>

    // Code coverage diff between base branch:release and head branch: bug/mustache-binding-extra-space 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :red_circle: | total | 51.94 **(-0.02)** | 33.62 **(-0.02)** | 29.99 **(0)** | 52.61 **(-0.02)**
 :red_circle: | app/client/src/utils/WidgetPropsUtils.tsx | 63.7 **(-0.17)** | 57.29 **(-0.03)** | 53.49 **(0)** | 62.93 **(-0.17)**
 :red_circle: | app/client/src/utils/migrations/TableWidget.ts | 65.91 **(-12.79)** | 55 **(-8.46)** | 68.75 **(-9.82)** | 68.91 **(-12.09)**
 :sparkles: :new: | **app/client/src/widgets/TableWidget/TableWidgetConstants.ts** | **100** | **50** | **100** | **100**</details>

### CODE REVIEW

**Review by riodeuno — APPROVED** [review]
Code looks good. @vivekverma2312 please verify the deploy preview.

**Comment by vivekverma2312** [comment]
@techbhavin 
1. Default value in computed value property for column is coming different from prod
Prod : `{{currentRow.email}}`
Deploy preview : `Table1.sanatizedTableData.map((currentRow) => ( currentRow.first_name ))`
2. Evaluated value for computed value property is also not proper
3. Changing binding in the computed property do not change the value in the column
https://www.loom.com/share/06cb3309bb5d434595997474f01bfb65

File: app/client/src/components/propertyControls/ComputeTablePropertyControl.tsx [hunk 1]
```diff
@@ -132,7 +133,7 @@ class ComputeTablePropertyControl extends BaseControl<
 
   getComputedValue = (value: string, tableId: string) => {
     const stringToEvaluate = stringToJS(value);
-    return `{{${tableId}.sanitizedTableData.map((currentRow) => { return ${stringToEvaluate}})}}`;
```
Comment (vicky-primathon) [inline app/client/src/components/propertyControls/ComputeTablePropertyControl.tsx:135]: @techbhavin  The existing table will have the values in this format 
```
Table1.sanitizedTableData.map((currentRow) => {
  return currentRow.id;
});
```
If we do this here without running migrations over existing table than computations will not work.

**Review by somangshu — DISMISSED** [review]

**Review by somangshu — APPROVED** [review]

**Comment by vivekverma2312** [comment]
@techbhavin @vicky-primathon Issue is fixed in the current deploy preview and can be merged into release.

**Comment by somangshu** [comment]
@techbhavin please merge latest release here

**Review by vicky-primathon — APPROVED** [review]

**Comment by somangshu** [comment]
All test failing here, Rerunning.

@techbhavin please merge latest release here

**Review by somangshu — APPROVED** [review]

**Review by somangshu — DISMISSED** [review]

**Comment by somangshu** [comment]
@techbhavin please check related test cases failing

**Review by somangshu — APPROVED** [review]
Approving to run tests

**Pull request merged**